diff --git a/tools/automate.py b/tools/automate.py
index 45a592d..8a8901a 100644
--- a/tools/automate.py
+++ b/tools/automate.py
@@ -864,7 +864,7 @@ def getenv():
     # GN configuration
     env["CEF_USE_GN"] = "1"
     # Issue #73 patch applied here with "use_allocator=none"
-    env["GN_DEFINES"] = "use_sysroot=true use_allocator=none symbol_level=1"
+    env["GN_DEFINES"] = "use_sysroot=false use_allocator=none symbol_level=1"
     # env["GN_DEFINES"] += " use_gtk3=false"
     # To perform an official build set GYP_DEFINES=buildtype=Official.
     # This will disable debugging code and enable additional link-time
