From 0c24e90b4448a1e34c4f6ac26deeb10935d33e60 Mon Sep 17 00:00:00 2001
From: Marshall Greenblatt <magreenblatt@gmail.com>
Date: Tue, 18 Jul 2017 17:07:23 -0400
Subject: [PATCH] Fix errors on gclient sync of older release branches (issue
 #2211)

---
 tools/automate/automate-git.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/automate/automate-git.py b/tools/automate/automate-git.py
index e345182b..c7357f66 100644
--- a/tools/automate/automate-git.py
+++ b/tools/automate/automate-git.py
@@ -896,8 +896,8 @@ if not os.path.exists(gclient_file) or options.forceconfig:
 # Initial Chromium checkout.
 if not options.nochromiumupdate and not os.path.exists(chromium_src_dir):
   chromium_checkout_new = True
-  run("gclient sync --nohooks --with_branch_heads --jobs 16", chromium_dir, \
-      depot_tools_dir)
+  run("gclient sync --nohooks --with_branch_heads --disable-syntax-validation --jobs 16", \
+      chromium_dir, depot_tools_dir)
 else:
   chromium_checkout_new = False
 
@@ -981,7 +981,7 @@ if chromium_checkout_changed:
   os.environ['GYP_CHROMIUM_NO_ACTION'] = '1'
 
   # Update third-party dependencies including branch/tag information.
-  run("gclient sync %s--with_branch_heads --jobs 16" % \
+  run("gclient sync %s--with_branch_heads --disable-syntax-validation --jobs 16" % \
       (('--reset ' if options.forceclean else '')), \
       chromium_dir, depot_tools_dir)
 
-- 
2.14.2

